### ISSUE TITLE
Fails to install in node:14-alpine

### ISSUE THREAD

**Issue by neilmayhew** [DoctorEvidence/lmdb-js#158]
Since 2.3.0:

```shellsession
$ docker run --rm -it --entrypoint=sh node:14-alpine
/ # npm install lmdb@2.3.0

> msgpackr-extract@1.1.4 install /node_modules/msgpackr-extract
> node-gyp-build-optional-packages


> lmdb@2.3.0 install /node_modules/lmdb
> node-gyp-build-optional-packages

gyp ERR! find Python
```

I'm not sure if this is the cause, but since 2.3.0 the prebuilds have disappeared from the package.

**Comment by kriszyp** [comment]
The prebuilds have moved to separate platform-specific packages so you don't have to have binaries for every platform installed. So `npm install` should have also installed lmdb-linux-x64 (unless you are using ARM), which should have node.napi.musl.node as the binary for your system. Can you verify that you have a node_modules/lmdb-linux-x64/node.napi.musl.node?

**Comment by kriszyp** [comment]
OK, I think I figured out the issue, looks like symbol resolution on alpine must not be lazy in the same way as glibc, so need a more specific nan-free napi build for alpine. I will get that released soon.

**Comment by neilmayhew** [comment]
OK, thanks.

FYI, I'm doing this in Docker (with the `node:14-alpine` image) so no packages are installed before I try to install `lmdb`. (`npm list` returns an empty result.) After trying to install `lmdb`, `/node_modules` is still empty.

**Comment by kriszyp** [comment]
I think that is probably because npm install will revert all the package installations if it fails. I was able to reproduce on `node:14-alpine` with `npm install lmdb --ignore-scripts --verbose && node -e "console.log(require('lmdb'))"` which gives the specific symbol resolution failure error that was causing the problem.

**Comment by neilmayhew** [comment]
I see that you have an Alpine build in the GitHub workflow. However, it's for node 12 and is only for aarch64. It would be great if you could have one for x86_64 as well, and with a newer version of node. However, I realize there are only so many hours in the day :-)

**Comment by neilmayhew** [comment]
Hah! You're too quick for me :-)

**Comment by kriszyp** [comment]
This is the one for alpine x64:
https://github.com/DoctorEvidence/lmdb-js/blob/master/.github/workflows/prebuild.yml#L95-L97
The problem was that NAPI build included NAN symbols, which works can lazily conditionally executed on other linux flavors, but apparently not on alpine, so need builds for NAPI and NAN versions separately.

**Comment by kriszyp** [comment]
Ok, published v2.3.2 with appropriate builds, wanna give that a try?

**Comment by neilmayhew** [comment]
Working perfectly! Thanks so much for your quick work on this.
